core/metacling/src/TCling.cxx

```diff
@@ -1859,6 +1860,13 @@ void TCling::UnRegisterTClassUpdate(const TClass *oldcl)
    }
 }
 
+
+////////////////////////////////////////////////////////////////////////////////
+/// Parse a conversion rule.
+Bool_t TCling::ParseClassConversionRule(const std::string &rule, MembersMap_t &result, std::string &error_string) {
```

Comment:
This might work as well as a free standing function ... or maybe we need to new abstract interface.